js/graph_spectrum_calc.js

```diff
@@ -24,15 +24,14 @@ var GraphSpectrumCalc = GraphSpectrumCalc || {
 };
 
 GraphSpectrumCalc.initialize = function(flightLog, sysConfig) {
-
     this._flightLog = flightLog; 
     this._sysConfig = sysConfig;
 
-    var gyroRate = (1000000 / this._sysConfig['looptime']).toFixed(0);
-    this._blackBoxRate = gyroRate * this._sysConfig['frameIntervalPNum'] / this._sysConfig['frameIntervalPDenom'];
-    if (this._sysConfig.pid_process_denom != null) {
-        this._blackBoxRate = this._blackBoxRate / this._sysConfig.pid_process_denom;
-    }
+    let minTime = this._flightLog.getMinTime();
+    let maxTime = this._flightLog.getMaxTime();
+    var allChunks = this._flightLog.getChunksInTimeRange(minTime, maxTime);
```

Comment:
```suggestion
    const minTime = this._flightLog.getMinTime();
    const maxTime = this._flightLog.getMaxTime();
    const allChunks = this._flightLog.getChunksInTimeRange(minTime, maxTime);
```

Reply:
Is it ok to make the same changes in the remaining functions of this js file in this pr? I also want to do some minor refactoring here, as it is best way to dive into project